ddtrace/filters.py

```diff
@@ -73,18 +70,3 @@ def process_trace(self, trace):
                     if regexp.match(url):
                         return None
         return trace
-
-
-class TraceCiVisibilityFilter(TraceFilter):
```

Comment:
This is a breaking change

Reply:
I am well aware of it. There is no way that anyone can use it as is outsize CI Visibility space. It should have been added to `internal` module in the first place.